src/native/libs/System.JavaScript/ReadMe.md

```diff
@@ -0,0 +1,15 @@
+# System.JavaScript
```

Comment:
All projects under src/native/libs are called `System.*.Native`. This naming convention looks different from what's in this directory so far.